plugin-ci/orb.yml

```diff
@@ -24,19 +24,35 @@ commands:
           from: << parameters.filename >>
           to: << parameters.bucket >>
           arguments: '--acl public-read --cache-control no-cache'
+  install-node:
+    description: Install node
+    steps:
+      - run:
+          name: Install Node
+          command: |
+            set +e
+            wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash
```

Comment:
nit: I wonder if we could make the nvm and default node versions variables?

```yaml
    parameters:
      nvm_version:
        type: string
        default: v0.35.1
      node_version:
        type: string
        default: 13.14
```

0/5 on that, I don't know if we can change those values from the orb user side so it might be unnecessary

Reply:
As for nvm version, I don't think that's a big deal to have that hardcoded